code/controllers/subsystem/persistence.dm

```diff
@@ -16,6 +16,7 @@ SUBSYSTEM_DEF(persistence)
 	var/list/obj/structure/sign/painting/painting_frames = list()
 	var/list/obj/item/storage/photo_album/photo_albums
 	var/list/paintings = list()
```

Comment:
Paintings needs to be protected var if it contains information about the price and owner

Reply:
Tried to slap the good ol'reliable "protection = CONFIG_ENTRY_LOCKED" but then i remember it's tied to configurations rather subsystems, so i'm kinda stumped on that one...